src/racer/nameres.rs

```diff
@@ -861,13 +861,21 @@ pub fn search_crate_root(
     namespace: Namespace,
     session: &Session,
     import_info: &ImportInfo,
+    skip: bool,
```

Comment:
Please use more concrete name, like `skip_this_file` or `skip_if_self_is_mod`.
But, I don't have a good idea now :slightly_frowning_face: , sorry.

Reply:
Me too :) So added a comment where it is used. Would it be better if we change the `modfpath` to an `Option`?

Reply:
Since we use `modfpath` to get parent directory we can't do that.
How about `skip_modfpath`?